### PR TITLE
[wip] [TEST-ONLY]  Fix failing perf tests

### PR DESCRIPTION
**Pull request by skonto**
**What this PR does / why we need it**:
Test failing performance tests.

### CODE REVIEW

**Comment by skonto** [comment]
```
+ oc wait '--for=jsonpath={.status.availableReplicas}=4' machineset --all -n openshift-machine-api --timeout=-1s
machineset.machine.openshift.io/ci-ocp-4-11-amd64-aws-td2x8-worker-us-east-1a condition met
machineset.machine.openshift.io/ci-ocp-4-11-amd64-aws-td2x8-worker-us-east-1b condition met
machineset.machine.openshift.io/ci-ocp-4-11-amd64-aws-td2x8-worker-us-east-1c condition met
machineset.machine.openshift.io/ci-ocp-4-11-amd64-aws-td2x8-worker-us-east-1f condition met
error: availableReplicas is not found
```
Scaling didnt work in this run.

**Comment by skonto** [comment]
Two tests still fail downstream even with the changes from the upstream:

```
2023/11/30 11:58:45 SLA 1 failed. P95 latency is not in 100-110ms time range: 111.338281ms
+ oc logs -n default job.batch/rollout-probe-activator-direct-lin

2023/11/30 12:01:59 SLA 1 failed. P95 latency is not in 100-110ms time range: 147.249505ms
+ oc logs -n default job.batch/rollout-probe-queue-direct
```

File: test/performance/performance-tests.sh [hunk 1]
```diff
@@ -204,6 +204,8 @@ kubectl wait --for=delete ksvc/load-test-200 --timeout=60s -n "$ns"
 ###############################################################################################
 header "Rollout probe: activator direct"
 
+toggle_feature scale-to-zero-grace-period 10s config-autoscaler
```
Comment (skonto) [inline test/performance/performance-tests.sh:207]: wrong place.

**Comment by ReToCode** [comment]
@skonto does this need more work or is it ok?

**Comment by skonto** [comment]
I will close it want to check something before that.

**Pull request closed without merging**